.ci/run.sh

```diff
@@ -2,7 +2,7 @@
 
 trap exit SIGINT
 
-python3 -m flake8 pacai
+python3 -m flake8 ../pacai --config=./flake8.cfg
```

Comment:
This will not work correctly if someone is running this from the root directory (which is what we do in the CI).
(Both the package (`../pacai`) and the path (`./flake8.cfg`) will not work.)
Instead, let's get the path of the directory this file lives in, then we can use that to properly locate the flake file.

We want to be able to invoke it from anywhere.
Something like:
```
#!/bin/bash

readonly THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"

trap exit SIGINT

python3 -m flake8 "${THIS_DIR}/../pacai" --config "${THIS_DIR}/flake8.cfg"
if [[ $? -eq 0 ]]; then
    echo "Style passed!"
else
    echo "Style failed. :("
fi
```